tests/integration/test_check_downloaded_output.py

```diff
@@ -23,7 +23,7 @@ def test_check_downloaded_output(test_env, default_request, tmpdir):
     * Check that dir app/ contains application source code
     * Check that the same full path filename is not duplicated
     """
-    response = default_request.complete_response
+    response = default_request["gomod"].complete_response
```

Comment:
Just out of curiosity, why are we testing this just for `gomod` and not for `npm`?

Reply:
That will be done later. For this particular test it's not a simple change because checks are specific for gomod bundle.

Reply:
@nirzari yes, @ejegrova is right. I created this PR to prepare for npm changes :)